docs/site/home.html

```diff
@@ -137,6 +137,9 @@ <h3 class="block-title">Examples</h3>
 
       <ul>
         <li>
+          <a href="https://github.com/GoogleCloudPlatform/nodejs-getting-started">Getting Started with Compute Engine and Node.js</a> (<em><a href="https://github.com/GoogleCloudPlatform/nodejs-getting-started">source</a></em>) - A tutorial for building a web application on Google Cloud Platform.
```

Comment:
Should both links go to Github..? Or should one go to https://cloud.google.com/nodejs/?